hack/tests/subcommand-bundle.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+source hack/lib/test_lib.sh
+
+function check_dir() {
+  if [[ $3 == 0 ]]; then
+    if [[ -d "$2" ]]; then
+      error_text "${1}: directory ${2} should not exist"
+      exit 1
+    fi
+  else
+    if [[ ! -d "$2" ]]; then
+      error_text "${1}: directory ${2} should exist"
+      exit 1
+    fi
+  fi
+}
+
+function check_file() {
+  if [[ $3 == 0 ]]; then
+    if [[ -f "$2" ]]; then
+      error_text "${1}: file ${2} should not exist"
+      exit 1
+    fi
+  else
+    if [[ ! -f "$2" ]]; then
+      error_text "${1}: file ${2} should exist"
+      exit 1
+    fi
+  fi
+}
```

Comment:
Nit: move these to `test_lib.sh`?

Reply:
Will do that in #2737 when I add validate tests.